pkg/operator/controller/ingress/deployment.go

```diff
@@ -47,11 +47,11 @@ const (
 
 	RouterHTTPHeaderNameCaseAdjustments = "ROUTER_H1_CASE_ADJUST"
 
-	RouterLogLevelEnvName        = "ROUTER_LOG_LEVEL"
-	RouterSyslogAddressEnvName   = "ROUTER_SYSLOG_ADDRESS"
-	RouterSyslogFormatEnvName    = "ROUTER_SYSLOG_FORMAT"
-	RouterSyslogFacilityEnvName  = "ROUTER_LOG_FACILITY"
-	RouterSyslogMaxLengthEnvName = "ROUTER_LOG_MAX_LENGTH"
+	RouterLogLevelEnvName       = "ROUTER_LOG_LEVEL"
+	RouterLogMaxLengthEnvName   = "ROUTER_LOG_MAX_LENGTH"
+	RouterSyslogAddressEnvName  = "ROUTER_SYSLOG_ADDRESS"
+	RouterSyslogFormatEnvName   = "ROUTER_SYSLOG_FORMAT"
+	RouterSyslogFacilityEnvName = "ROUTER_LOG_FACILITY"
```

Comment:
If you haven't actually changed anything here, please revert the reordering.  It makes backports more difficult over time.

Reply:
I thought we discussed having two environment parameters, one for syslog length and one for sidecar log length?

Reply:
yeap, the modification I did is exactly for that, the RouterLogMaxLengthEnvName will be the name of variable for both cases, Syslog and Container thus we will have just one var

Reply:
Now we have two different inputs for syslog length and sidecar length, and two different places  a single environment variable is set: https://github.com/openshift/cluster-ingress-operator/pull/900/files#diff-03760f03eeac1174e38e1ec5fecadcab4cb38d05e00509f201d7ae64dc1c7c36R767-R790

It's possible that one of these settings will overwrite the other if they are different values.

Reply:
The two assignments in https://github.com/openshift/cluster-ingress-operator/pull/900/files#diff-03760f03eeac1174e38e1ec5fecadcab4cb38d05e00509f201d7ae64dc1c7c36R767-R790 are for two different `case` statements in the same `switch` statement:
```go
switch {
case accessLogging.Destination.Type == operatorv1.ContainerLoggingDestinationType:
	// [...]
	// [Set ROUTER_LOG_MAX_LENGTH to accessLogging.Destination.Container.MaxLength.]
	// [...]
case accessLogging.Destination.Type == operatorv1.SyslogLoggingDestinationType:
	// [...]
	// [Set ROUTER_LOG_MAX_LENGTH to accessLogging.Destination.Syslog.MaxLength.]
	// [...]
}
```
I think it should be fine.  Am I missing anything?

Reply:
No, it was me.  I overlooked the case statement - https://github.com/openshift/cluster-ingress-operator/pull/900#discussion_r1230061006